src/main/webapp/WEB-INF/tags/shared/feedbackResponseComment.tag

```diff
@@ -132,127 +134,117 @@
                                        <c:if test="${frc.showGiverNameToResponseGiver}">checked="checked"</c:if>>
                             </td>
                         </tr>
-                        <c:if test="${frc.responseVisibleToRecipient}">
```

Comment:
I didn't put these tests in the tag for no reason. Check instructorFeedbackResultsByGRQ or ByRGQ ("those things") to see what I mean.